.github/workflows/build_pr_documentation.yml

```diff
@@ -42,7 +34,6 @@ jobs:
         with:
           repository: 'huggingface/optimum'
           path: optimum
-          ref: ${{ github.event.pull_request.merge_commit_sha }}
```

Comment:
do you mean that this SHA is main latest commit SHA instead of a potential merge commit sha?

Reply:
No, by default the ref will point to the commit SHA of the event that triggered the workflow

Reply:
I don't understand. `github.event.pull_request.merge_commit_sha` should be the merge commit from a PR, is that not? Why is the documentation built against main?

Reply:
I don't know the details, but for sure it doesn't take into account the current PR changes to the workflow. See this PR for many examples: https://github.com/huggingface/optimum/pull/1624